frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SharingPaneButton/SharingPaneButton.tsx

```diff
@@ -20,32 +24,55 @@ export const SharingPaneButton = ({
   title,
   onClick,
   badge,
-  externalLink = false,
   "data-testid": dataTestId,
-}: SharingOptionProps) => (
-  <Paper
-    className={S.Container}
-    p={24}
-    pt={52}
-    withBorder
-    data-testid={dataTestId}
-    onClick={onClick}
-    mih="100%"
-    pos="relative"
-    w={"22rem"}
-  >
-    <Stack>
-      {externalLink && (
-        <Box pos="absolute" top={12} right={12}>
-          <Icon name="share" color="var(--external-link-icon-color)" />
+  isDisabled,
+  disabledLink,
+}: SharingOptionProps) => {
+  return (
+    <Paper
+      component="article"
+      className={cx(S.Container, { [S.Disabled]: isDisabled })}
+      p={24}
+      pt={52}
+      withBorder
+      data-testid={dataTestId}
+      onClick={onClick}
+      mih="100%"
+      pos="relative"
+      w={"22rem"}
+      aria-label={title}
+    >
+      <Stack pb={isDisabled ? "md" : undefined}>
+        <Center mb={32}>{illustration}</Center>
+        <Group align="center" spacing="sm">
+          <Title size="h2">{title}</Title>
+          {badge}
+        </Group>
+        {children}
+      </Stack>
+      {isDisabled && (
+        <Box
+          pos="absolute"
+          bottom={0}
+          left={0}
+          w="100%"
+          bg="var(--mb-color-background-disabled)"
+        >
+          <Text
+            c="var(--mb-color-text-secondary)"
+            weight="bold"
+            align="center"
+            py="sm"
+            lh="1"
+          >
+            {t`Disabled.`}{" "}
```

Comment:
do we need that extra space? I'd rather use a padding value or something like that

Reply:
I'm cool either way. We seem to use `{" "}` in many places by the way, so it's not unordinary. I'll check if I use different method for this kind of spacing in a different place in this PR.

Reply:
Fixed in 004d246310f7ee1f924d2b0808400dda12d2f8c5